### ISSUE TITLE
Database error 1 while preparing stmt: SQL logic error

### ISSUE THREAD

**Issue by l29ah** [markfasheh/duperemove#333]
```
∞ duperemove -drq --hashfile=/var/db/duperemove.hash -h /                                 
dbfile_open_handle()/20170: Database error 1 while preparing stmt: SQL logic error
```
duperemove 0.14.1

**Comment by JackSlateur** [comment]
Hello,
Was your hashfile created with an older version ?
0.14.1 is not compatible with hashfile created with versions older than 0.14

**Comment by l29ah** [comment]
Perhaps. https://github.com/markfasheh/duperemove/issues/291 suggest it would have been recreated in such case.

**Comment by hhyyrylainen** [comment]
While switching from 0.13 to latest commit I had to delete all my hash files to get past this error.

**Comment by JackSlateur** [comment]
Hello,

Thank you for the report
I believe this is fixed: 5c0cada6d079921cc96fa705fd79992d59ff09ea

Best regards,
